examples/README.md

```diff
@@ -0,0 +1,189 @@
+# Examples
+
+These gifs were created using [Charm VHS](https://github.com/charmbracelet/vhs).
+
+## Barchart ([barchart.rs](./barchart.rs)
+
+```shell
+cargo run --example=barchart --features=crossterm
+```
+
+![Barchart][barchart.gif]
+
+## Block ([block.rs](./block.rs))
+
+```shell
+cargo run --example=block --features=crossterm
+```
+
+![Block][block.gif]
+
+## Calendar ([calendar.rs](./calendar.rs))
+
+```shell
+cargo run --example=calendar --features=crossterm widget-calendar
+```
+
+![Calendar][calendar.gif]
+
+## Canvas ([canvas.rs](./canvas.rs))
+
+Note: VHS renders braille dots incorrectly - see <https://github.com/charmbracelet/vhs/issues/322>
+
+```shell
+cargo run --example=canvas --features=crossterm
+```
+
+![Canvas][canvas.gif]
+
+## Chart ([chart.rs](./chart.rs))
+
+Note: VHS renders braille dots incorrectly - see <https://github.com/charmbracelet/vhs/issues/322>
+
+```shell
+cargo run --example=chart --features=crossterm
+```
+
+![Chart][chart.gif]
+
+## Custom Widget ([custom_widget.rs](./custom_widget.rs))
+
+```shell
+cargo run --example=custom_widget --features=crossterm
+```
+
+![Custom Widget][custom_widget.gif]
+
+## Gauge ([gauge.rs](./gauge.rs))
+
+TODO Gauges render poorly on a tty
```

Comment:
Seems fine to me, only thing is the last gauge is missing borders but can be fixed via:

```diff
diff --git a/examples/gauge.rs b/examples/gauge.rs
index 5423ca7..26773d6 100644
--- a/examples/gauge.rs
+++ b/examples/gauge.rs
@@ -154,7 +154,7 @@ fn ui<B: Backend>(f: &mut Frame<B>, app: &App) {
 
     let label = format!("{}/100", app.progress4);
     let gauge = Gauge::default()
-        .block(Block::default().title("Gauge4"))
+        .block(Block::default().title("Gauge4").borders(Borders::ALL))
         .gauge_style(
             Style::default()
                 .fg(Color::Cyan)

```

Reply:
From my terminal:
<img width="816" alt="image" src="https://github.com/tui-rs-revival/ratatui/assets/381361/45a35ad0-82ff-4895-92a0-1bf04341f5f7">

From the image:
![image](https://github.com/tui-rs-revival/ratatui/assets/381361/fe03a2f4-a2a4-4f2c-860c-76e115eea506)

Reply:
Added the border to this.